zScanner/Views/NewDocument/PhotoSelectorCollectionViewCell.swift

```diff
@@ -86,14 +94,15 @@ class PhotoSelectorCollectionViewCell: UICollectionViewCell {
         return image
     }()
     
-    private var textField : UITextField = {
+    private lazy var textField : UITextField = {
```

Comment:
YOu don't need to use lazy on UIView, because the view is already created lazily. This is needed only on UIViewController, as the view, it's needed later than the instance of the View Controller itself. 

There is no obvious harm in doing that. If you take a look (or put a breakpoint) you will see it's called directlu in the init (trough `setupView()` ) and I believe it may slow down the compiler a little bit. if you use it only ince there is no issue, but if you'll have thousands of usages it will make the compile time longer. 

Again, this is just pointing out my experience and giving you more context. I'm perfectly fine with keeping the lazy keyword.

Reply:
I also thought I didn't need to put it like lazy, but Xcode said that I can't use subscribe for the textfield without it. I was a bit confused.

Reply:
Good point, thank you.